### PR TITLE
[Merged by Bors] - chore(Order): add missing `inst` prefix to instance names

### PR DESCRIPTION
**Pull request by eric-wieser**
This is not exhaustive; it largely does not rename instances that relate to algebra, and only focuses on the "core" order files.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Order/Kleene.lean
Comment (YaelDillies) [inline Mathlib/Algebra/Order/Kleene.lean:327]: Now that instances are not protected by default (and have names that let them stay unprotected), I think we should move towards unqualifying the names
```suggestion
  { semiring, instSemilatticeSup, instOrderBot with
```

Comment (eric-wieser) [reply Mathlib/Algebra/Order/Kleene.lean:327]: I'd rather leave that to a follow-up; it's much easier to do a replacement of  `Foo.instBar` with `instBar` via a regex later.

It's also not fully clear to me whether unprotected instances is the right choice.

File: Mathlib/Analysis/Normed/Order/Lattice.lean
Comment (YaelDillies) [inline Mathlib/Analysis/Normed/Order/Lattice.lean:112]: That's technically scope-creep wrt your PR description, but I'm fine with it

**Review by YaelDillies — APPROVED** [review]
Thanks!

maintainer merge

File: Mathlib/Analysis/Normed/Order/Lattice.lean
Comment (eric-wieser) [reply Mathlib/Analysis/Normed/Order/Lattice.lean:112]: Good catch, after changing this one I realized I was taking on too much at once.

**Comment by mattrobball** [comment]
bors merge

**Pull request closed without merging**
